docs/table-of-contents.md

```diff
@@ -78,47 +196,14 @@ project:
 
 Note that when these files are excluded, they can still be specifically referenced by other files in your project (e.g. in an {myst:directive}`include directives <include>` or as a download), however, a change in those files will not trigger a build. An alternative in this case is to generate a table of contents (see [](./table-of-contents.md)). By default hidden folders (those starting with `.`, like `.git`), `_build` and `node_modules` are excluded.
 
-(toc-format)=
-
-## Defining a `_toc.yml` using Jupyter Book’s format
```

Comment:
I am assuming that we haven't removed this functionality, we could put this into a dropdown under "Compatibility with JupyterBook", and keep it around until we fully deprecate that path.